tests/test_files/bundles/verification/ui.invalid.bundle.yaml

```diff
@@ -39,12 +39,12 @@ data:
                 }
               
             ]
-          categories: openshift required
```

Comment:
Won't these already fail?

Reply:
Yes, just wanted to be consistent with the invalid values